extension/ezoe/modules/ezoe/classes/SpellChecker.php

```diff
@@ -12,10 +12,17 @@ class SpellChecker {
 	 *
 	 * @param $config Configuration name/value array.
 	 */
-	function SpellChecker(&$config) {
+	function __construct(&$config) {
 		$this->_config = $config;
 	}
 
+    /**
+     * @deprecated Use SpellChecker::__construct() instead
+     * @param $config
+     */
+	function SpellChecker(&$config) {
+		self::__construct($config);
```

Comment:
CS (same for all others it seems)

Reply:
hmm, this is already issue in the file from before from tinymce source

then ignore, but the docblock seems off here compared to other methods